tests/misc/projects/Issue8176/.gitignore

```diff
@@ -0,0 +1 @@
+/main.js
```

Comment:
All files generating into `bin/` are automatically ignored (e.g. `-js bin/main.js`)

Reply:
Good to know :)